src/serverenvironment.h

```diff
@@ -243,7 +244,10 @@ class ServerEnvironment : public Environment
 		-------------------------------------------
 	*/
 
-	ServerActiveObject* getActiveObject(u16 id);
+	ServerActiveObject* getActiveObject(u16 id)
```

Comment:
Needs style correction

Reply:
yes, code style will be fixed after functional review